internal/storage/types/location_config.go

```diff
@@ -143,6 +159,25 @@ func (c URLConfig) OpenBucket(ctx context.Context) (*blob.Bucket, error) {
 	if err != nil {
 		return nil, fmt.Errorf("open bucket by URL: %v", err)
 	}
+	return b, nil
+}
+
+type SSConfig struct {
+	SS string `json:"url"`
+}
```

Comment:
`SS` isn't a great name for the URL attribute. Here's an alternative where I also break down the configuration into a few fields:

```go
type SSConfig struct {
	URL string `json:"url"`
	Username string `json:"username"`
	APIKey string `json:"api_key"`
}
```

For example:

```json
{
  "url": "http://127.0.0.1:12345",
  "username": "test",
  "api_key": "test"
}
```

I think in the past we've encoded the credentials in the URL, e.g. `http://test:test@127.0.0.1:12345` but unsure that's a good idea in this case given that we'll likely want to manage secrets externally at some point. That's why I think it's better to break it down like this.